Unreal/SystemLocalized/det/IpDrv.det

```diff
@@ -5,8 +5,8 @@ Object=(Name=IpDrv.CompressCommandlet,Class=Class,MetaClass=Core.Commandlet)
 Object=(Name=IpDrv.DecompressCommandlet,Class=Class,MetaClass=Core.Commandlet)
 Object=(Name=IpDrv.TcpNetDriver,Class=Class,MetaClass=Engine.NetDriver)
 Object=(Name=IpDrv.UdpBeacon,Class=Class,MetaClass=Engine.Actor)
-Preferences=(Caption="TCP/IP Network Play",Parent="Networking",Class=IpDrv.TcpNetDriver)
-Preferences=(Caption="Server Beacon",Parent="Networking",Class=IpDrv.UdpBeacon,Immediate=True)
+Preferences=(Caption="TCP/IP Network Play",Parent="Networking",LangCaption="TCP/IP-Netzwerkspiel",Class=IpDrv.TcpNetDriver)
+Preferences=(Caption="Server Beacon",Parent="Networking",LangCaption="Server-Broadcast",Class=IpDrv.UdpBeacon,Immediate=True)
```

Comment:
```suggestion
Preferences=(Caption="Server Beacon",Parent="Networking",LangCaption="Server-Beacon",Class=IpDrv.UdpBeacon,Immediate=True)
```
Translating an English term with a completely different English term would be very misleading and these seem to have nothing to do with one another.